packages/flutter/test/material/navigation_rail_test.dart

```diff
@@ -3631,6 +3632,62 @@ void main() {
     expect(tester.takeException(), isNull);
   });
 
+  testWidgets('NavigationRail labels shall not overflow if longer texts provided - extended', (WidgetTester tester) async {
+
+    // The navigation rail has a narrow width constraint. The text should wrap.
```

Comment:
Consider adding a regression comment such as:
```suggestion
  testWidgets('NavigationRail labels shall not overflow if longer texts provided - extended', (WidgetTester tester) async {
    // Regression test for https://github.com/flutter/flutter/issues/110901.
    // The navigation rail has a narrow width constraint. The text should wrap.
```